os/kernel/semaphore/sem_post.c

```diff
@@ -211,6 +211,7 @@ int sem_post(FAR sem_t *sem)
 		saved_state = irqsave();
 
 		/* Perform the semaphore unlock operation. */
+		ASSERT_INFO(sem->semcount < SEM_VALUE_MAX, "sem = 0x%x, sem->semcount = %d, SEM_VALUE_MAX = %d", sem);
```

Comment:
`semcount` and `SEM_VALUE_MAX` are unnecessary to debug this case. The sem address and caller are necessary. In this case, we should know what sem is and who calls like this.

Reply:
updated